### ISSUE TITLE
unlocks 30% more memory?

### ISSUE THREAD

**Issue by SheidCraft** [Electry/VitaGrafixPatchlist#73]
In firmware 3.50 there was an opportunity to increase the amount of RAM by 30% with limitation of multimedia capabilities of the console. But this only worked in games such as Borderlands 2, World of Final Fantasy, Resident Evil REVELATENS 2 and in the latest Minecraft updates, if I'm not mistaken. And so I wonder if it’s possible to do something similar in games like Assassins Creed, Spider Man or Jack and Dexter?

**Comment by justalex95** [comment]
Electry did this with the amazing spiderman, I was thinking if it was possible with killzone or need for speed it would be brilliant, better rresolutions for need for speed or solid 60 fps in killzone could be possible, but it might be game dependant or just not pheaseable.

I think some games assign memory dynamically with a set upper limit, killzone might do this. In which case, it cant be changed, something like that anyways

If electry can do it he will

**Comment by SheidCraft** [comment]
@justalex95 Strange... I ran Spider Man in 544p resolution, but the trophies worked quietly.  Maybe me need to write something else in config.txt?

**Comment by Electry** [comment]
Yes, there is a flag in PARAM.SFO that makes extra 109 MB available (256 MB + 109 MB = total of 365 MB LPDDR2 RAM), but unless a game is specifically designed to take advantage of extra memory I don't see how it would help.

VRAM (also called CDRAM) capacity is the limitation in most cases. Unfortunately you always only get 112 MB, no matter what. The remaining 16 MB is used by SceShell/LiveArea.

**Comment by Fershock** [comment]
Hi @Electry , one question, have you thought about working on the Native Resolution for PSP games on PSVITA project?  I think you are one of the most talented people in the psvita scene and I think you could do it.  Also if you complete it they give you a remuneration of 700 dollars
